proto/zepben/protobuf/cim/iec61970/base/core/Name.proto

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+
+syntax = "proto3";
+option java_multiple_files = true;
+option java_package = "com.zepben.protobuf.cim.iec61970.base.core";
+option csharp_namespace = "Zepben.Protobuf.CIM.IEC61970.Base.Core";
+package zepben.protobuf.cim.iec61970.base.core;
+
+import "zepben/protobuf/cim/iec61970/base/core/IdentifiedObject.proto";
+import "zepben/protobuf/cim/iec61970/base/core/NameType.proto";
+
+/**
+ * The Name class provides the means to define any number of human readable names for an object. A name is **not** to
+ * be used for defining inter-object relationships. For inter-object relationships instead use the object
+ * identification 'mRID'.
+ */
+message Name {
+
+  /**
+   * Any free text that name the object.
+   */
+  string name = 1;
+
+  /**
+   * Type of this name.
+   */
+  NameType type = 2;
```

Comment:
I don't think this should be a NameType. It means every identified object that comes back with a name will be sent with a full NameType. We don't want this. This hsould probably just be the name type name.